app/src/main/java/com/nextcloud/talk/adapters/messages/IncomingTextMessageViewHolder.kt

```diff
@@ -103,8 +102,7 @@ class IncomingTextMessageViewHolder(itemView: View, payload: Any) :
 
         val messageParameters = message.messageParameters
         if (
-            (messageParameters == null || messageParameters.size <= 0) &&
-            TextMatchers.isMessageWithSingleEmoticonOnly(message.text)
```

Comment:
This is to check if the text contains of emoji only. If yes, the text size is increased.
By deleting this check, the text size of messages is always too huge.